src/Storage/Listener/LocalListener.php

```diff
@@ -29,7 +27,7 @@ class LocalListener extends AbstractListener {
  * @var array
  */
 	protected $_defaultConfig = [
-		'pathBuilder' => 'BasePath',
+		'pathBuilder' => 'Local',
```

Comment:
I think the default configuration for another adapters could be done inside constructor of descedant classes so clients could just use properly configured dedicated objects out of the box. It's more OOP for my taste.